components/spicedb/schema/schema.yaml

```diff
@@ -79,6 +82,25 @@ schema: |-
     permission write_info = editor + org->owner + org->installation_admin
     permission delete = editor + org->owner + org->installation_admin
   }
+
+  definition workspace {
+    relation org: organization
+    // The user that created the workspace
+    relation owner: user
+
+    // Whether a user can access a workspace (with an IDE)
+    //+ (hasAccessToRepository && isPrebuild) + everyoneIfIsShared
```

Comment:
This is missing, and will be added with a follow-up PR.

Reply:
We just discussed with Sven and Anton: This will break "prebuild access" (for everybody except owners) and "workspace sharing" everywhere we have the FGA feature flag enabled, namely preview environments.